cdhweb/people/admin.py

```diff
@@ -2,7 +2,7 @@
 from mezzanine.core.admin import DisplayableAdmin
```

Comment:
this entire file can be deleted since we're no longer managing the people admin through mezzanine/django. take a look at `wagtail_hooks.py` in the people app and there should be a TODO there about adding inline editing for relatedlinks, and have a look at the docs for [wagtail's `ModelAdmin`](https://docs.wagtail.io/en/stable/reference/contrib/modeladmin/index.html).

Reply:
addressed by 4a10cd4